src/commands/appSettings/filterDownloadAppSettings.ts

```diff
@@ -11,42 +11,75 @@ import { localize } from "../../localize";
 export async function filterDownloadAppSettings(context: IActionContext, sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationName: string): Promise<void> {
     let suppressPrompt: boolean = false;
     let overwriteSetting: boolean = false;
+    let suppressPromptOfSettingsToIgnore = false;
+    let overwriteSettingsToIgnore: boolean = false;
 
     const addedKeys: string[] = [];
     const updatedKeys: string[] = [];
     const userIgnoredKeys: string[] = [];
     const matchingKeys: string[] = [];
+    const securitySettingsIgnored: string[] = [];
+    const listOfSettingsToIgnore: string[] = ["AzureWebJobsStorage", "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING", "WEBSITE_CONTENTSHARE"];
 
     for (const key of Object.keys(sourceSettings)) {
-        if (destinationSettings[key] === undefined) {
-            addedKeys.push(key);
-            destinationSettings[key] = sourceSettings[key];
-        } else if (destinationSettings[key] === sourceSettings[key]) {
-            matchingKeys.push(key);
-        } else if (sourceSettings[key]) { // ignore empty settings
-            if (!suppressPrompt) {
+        if (listOfSettingsToIgnore.includes(key)) {
+            if (!suppressPromptOfSettingsToIgnore) {
                 const yesToAll: vscode.MessageItem = { title: localize('yesToAll', 'Yes to all') };
                 const noToAll: vscode.MessageItem = { title: localize('noToAll', 'No to all') };
-                const message: string = localize('overwriteSetting', 'Setting "{0}" already exists in "{1}". Overwrite?', key, destinationName);
+                const message: string = localize('overwriteSetting', 'Setting "{0}" has been identified as unsafe. Download?', key);
                 const result: vscode.MessageItem = await context.ui.showWarningMessage(message, { modal: true }, DialogResponses.yes, yesToAll, DialogResponses.no, noToAll);
                 if (result === DialogResponses.yes) {
-                    overwriteSetting = true;
+                    overwriteSettingsToIgnore = true;
                 } else if (result === yesToAll) {
-                    overwriteSetting = true;
-                    suppressPrompt = true;
+                    overwriteSettingsToIgnore = true;
+                    suppressPromptOfSettingsToIgnore = true;
```

Comment:
should the `suppressPromptOfSettingsToIgnore` to `true` for both `yesToAll` and `noToAll` ?

Reply:
Agreed. I think the piece of code that you suggested to replace it with in one of the below comments takes care of that now.